Acquisition/Utilities/DataGenerator/test/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+#@author S. V. Paulauskas
+#add_executable(unittest-WalkCorrector unittest-WalkCorrector.cpp ../source/WalkCorrector.cpp)
+#target_link_libraries(unittest-WalkCorrector UnitTest++ ${LIBS})
+#install(TARGETS unittest-WalkCorrector DESTINATION bin/unittests)
```

Comment:
It's all comments?

Reply:
Let's put in ticket for it.

Reply:
I'm not sure why that's not being built.

Reply:
Issue opened to resolve #42.

Reply:
More precisely, I opened #42 to resolve this issue.